egov-dss-dashboards/dashboard-analytics/ChartApiConfig.json

```diff
@@ -30854,5 +30854,104 @@
     "filterKeys": [{"key": "village"}],
     "insight": { },
     "_comment": " "
+  },
+  "locationNotEnabledByDistrict" : {
+    "chartName": "DSS_HEALTH_LOCATION_NOT_ENABLED_DISTRICT",
+    "queries": [
+      {
+        "module": "COMMON",
+        "dateRefField": "Data.household.auditDetails.createdTime",
+        "requestQueryMap": "{\"province\":\"Data.boundaryHierarchyCode.province.keyword\"}",
+        "indexName": "household-index-v1",
+        "aggrQuery": "{\"size\":0,\"aggs\":{\"FILTER\":{\"filter\":{\"bool\":{\"must_not\":{\"exists\":{\"field\":\"Data.geoPoint\"}}}},\"aggs\":{\"Location Not Enabled\":{\"terms\":{\"field\":\"Data.boundaryHierarchyCode.district.keyword\",\"size\":500}}}}}}"
+      }
+    ],
+    "chartType": "line",
+    "isCumulative": false,
+    "valueType": "number",
+    "drillChart": "locationNotEnabledByAP",
+    "documentType": "_doc",
+    "action": "",
+    "aggregationPaths": [
+      "Location Not Enabled"
+    ],
+    "filterKeys": [{"key": "district"}],
+    "insight": { },
+    "_comment": " "
+  },
+  "locationNotEnabledByAP" : {
+    "kind" :"drillDown",
+    "chartName": "DSS_HEALTH_LOCATION_NOT_ENABLED_AP",
+    "queries": [
+      {
+        "module": "COMMON",
+        "dateRefField": "Data.household.auditDetails.createdTime",
+        "requestQueryMap": "{\"province\":\"Data.boundaryHierarchyCode.province.keyword\",\"district\":\"Data.boundaryHierarchyCode.district.keyword\"}",
+        "indexName": "household-index-v1",
+        "aggrQuery": "{\"size\":0,\"aggs\":{\"FILTER\":{\"filter\":{\"bool\":{\"must_not\":{\"exists\":{\"field\":\"Data.geoPoint\"}}}},\"aggs\":{\"Location Not Enabled\":{\"terms\":{\"field\":\"Data.boundaryHierarchyCode.administrativeProvince.keyword\",\"size\":500}}}}}}"
```

Comment:
Can you recheck the query once? is it terms with field property or exists with field